src/rosdiscover/models/spawn_model.py

```diff
@@ -6,6 +6,7 @@
 
 from .plugins.gazebo import GazeboPlugin
 from ..interpreter import model
+import re
```

Comment:
Same thing with imports, sorry. Standard library imports go at the top.

Reply:
Done